OMCompiler/Compiler/SimCode/SerializeSparsityPattern.mo

```diff
@@ -0,0 +1,139 @@
+/*
+ * This file is part of OpenModelica.
+ *
+ * Copyright (c) 1998-CurrentYear, Open Source Modelica Consortium (OSMC),
```

Comment:
You need to add the current year here I think.

Reply:
I have no idea. I just copied that from another file.

Reply:
That's how the license looks, it's like that everywhere.